src/core/linalg/src/sparse/4C_linalg_fevector.hpp

```diff
@@ -233,6 +233,12 @@ namespace Core::LinAlg
       return vector_->Export(A, Importer.get_epetra_import(), CombineMode, Indexor);
     }
 
+    int export_to(const Epetra_SrcDistObject& A, const Core::LinAlg::Export& Exporter,
+        Epetra_CombineMode CombineMode, const Epetra_OffsetIndex* Indexor = nullptr)
```

Comment:
I don't understand, why the name `Indexor` is chosen (here and at other places in this file).